src/sql/workbench/browser/modelComponents/inputbox.component.ts

```diff
@@ -142,6 +143,14 @@ export default class InputBoxComponent extends ComponentBase implements ICompone
 	public validate(): Thenable<boolean> {
 		return super.validate().then(valid => {
 			this.inputElement.validate();
+
+			// set aria label based on validity of input
+			if (valid) {
```

Comment:
valid is being set by link 145?

Reply:
Looks like valid should end up being the combination of 144 and 145s results

Reply:
Agree with @Benjin, I'll test this a bit more.